src/main/java/com/google/api/generator/engine/writer/ImportWriterVisitor.java

```diff
@@ -174,6 +177,20 @@ public void visit(TryCatchStatement tryCatchStatement) {
     tryCatchStatement.catchVariableExpr().accept(this);
     statements(tryCatchStatement.catchBody());
   }
+  /** =============================== COMMENT =============================== */
+
+  @Override
+  public String visit (LineComment lineComment){
```

Comment:
Why is the return type String?

Reply:
it should be void, and let javaWriterVisitor to write the comment to String. Done Thanks.